src/main/groovy/org/transmartproject/batch/concept/ConceptTree.groovy

```diff
@@ -40,6 +40,7 @@ class ConceptTree {
 
     void loadExisting(Collection<ConceptNode> nodes) {
         nodes.each { n ->
+            assert !n.new
```

Comment:
@cataphract I don't think so. IMHO neither of the errors would help an end user much to fix the problem. Speaking about a developer the proposed message makes several assumption about existing code over there. Assumptions that would be not true under certain conditions. e.g. `loadExisting` would be misused by passing newly created nodes.